geoapps/inversion/components/data.py

```diff
@@ -161,6 +161,28 @@ def _initialize(self) -> None:
         self.survey, _, _ = self.create_survey()
         self.save_data(self.entity)
 
+    def pseudo_locations(self, locations: np.ndarray) -> np.ndarray:
```

Comment:
Maybe we should rename to avoid confusion with the dc definition where z is a function of pole/dipole spacings?  How about drape_locations?

Reply:
sounds good